unittest/python/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+SET(${PROJECT_NAME}_PYTHON_TESTS
+  bindings
+  #bindings_com
+  #bindings_dynamics
+  bindings_force
+  bindings_frame
+  bindings_geometry_object
+  bindings_geometry_object_urdf
+  bindings_inertia
+  bindings_joint_composite
+  bindings_motion
+  bindings_SE3
+  #explog
+  model
```

Comment:
Why are you not testing bindings_com, bindings_dynamics and explog?
Also, there is a script called tests which runs all Python tests one after the other. Do we still need it?

Reply:
As I said in the commit message:
> 3 are not passing, they are commented for now

But obviously we have to fix them :)

Reply:
About the `tests.py` script, I don't think we need it. It is easier to run those tests with CTests, as it is already integrated in the test suite, and giving each test individually to the system helps to run them in parallel, and to easily point which one is failing

Reply:
I hadn't seen the commit message. But that's weird, I was sure they were working. I am going to check this, but are they not passing at all, or just for a specific system, like 18.04 ?

Reply:
@gabrielebndn Can you take in charge the fixing of the Python tests?

Reply:
It is not about a specific system. The missing `;` was clearly making this file invalid.

Reply:
So, I tested these example both on my Mac (OSX El Capitan, 10.11) and on Ubuntu 16.04, current devel branch.
`explog` is passing on both systems.
`bindings_com` and `bindings_dynamics` seem to return correctly on Ubuntu 16.04, but they are triggering assertion errors on my Mac, where assertions are enabled (haven't tested enabling assertions on Ubuntu, I will do it soon).

Reply:
> It is not about a specific system. The missing `;` was clearly making this file invalid.

I was not in the right conversation thread, sorry, I misunderstood. I can try to test them again, I don't remember.

Reply:
I confirm `bindings_com` and `bindings_dynamics` raise an assertion, while `explog` segfaults in Debug mode. I think we can go on with this PR. I will open a separate issue on this